clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/dataMock.scala

```diff
@@ -39,6 +41,7 @@ case class ModelMockIndex(
     implicitly[FieldMapper[MetadataType]]
   override val queryInputMapper: FieldMapper[ModelMockQueryInput] =
     implicitly[FieldMapper[QueryInputType]]
+  override val keyFieldNames: Seq[String] = Seq("mock_key_long", "mock_key_string")
```

Comment:
Note that this stuff isn't a mock, it's test data. A mock is a stubbed/faked implementation of an object or API. This, on the other hand, is real object data and is therefore not a mock.